### ISSUE TITLE
Firefox "Save to pocket" functionality logged off

### ISSUE THREAD

**Issue by WolfganP** [gorhill/uBlock#3705]
### Describe the issue
Since a few FF versions ago (I'm currently in v60b11), I noticed the "Save to pocket" button (https://support.mozilla.org/en-US/kb/save-web-pages-later-pocket-firefox) appears always as logged off.

I applied all the suggestions as per https://help.getpocket.com/article/1046-trouble-saving-to-pocket-stuck-in-a-loop and nothing seems to work. I'm suspecting of 1st party isolation but I couldn't find a way to troubleshoot it (nothing related on uBlock log or FF Dev tools). 

I blindly tried a custom rule like "getpocket.com * 1p-script allow" but doesn't seem to work.

### Screenshot in which the issue can be seen

![ff-pocket-button](https://user-images.githubusercontent.com/2248211/38679485-631a90bc-3e53-11e8-8ada-3cf726ea72b3.jpg)

### Steps for anyone to reproduce the issue

1. Navigate to getpocket.com and verify you're logged in.
2. Go to any web page, ie cnn.com
3. Click the "Save to pocket" button on the omnibar, the function acts as if logged off.

### Your settings

- OS/version: Windows 10
- Browser/version:  Firefox v60b11
- uBlock Origin version: v1.15.24

**Comment by gorhill** [comment]
As always, first step is to find out what the logger says, it tells you if uBO blocks something which might be causing the issue.

**Comment by gorhill** [comment]
Closing this, as nowhere in the description is the case made that uBO is the issue.

You do say:

> nothing related on uBlock log

If so then why:

> I blindly tried a custom rule like "getpocket.com * 1p-script allow" but doesn't seem to work.

If uBO is blocking nothing as per logger, then there is no point blindly trying to create `allow` rules, something which is meaningful only in advanced user mode by the way.

I gave a try to Pocket in FF 59.0.2, and all worked just fine.
